components/kyma-metrics-collector/pkg/process/process.go

```diff
@@ -416,6 +459,16 @@ func (p *Process) populateCacheAndQueue(runtimes *kebruntime.RuntimesPage) {
 			}
 			continue
 		}
+
+		// record kebFetchedClusters metric for not trackable clusters
+		recordKEBFetchedClusters(
+			false,
```

Comment:
Please create a new const and use it here:

```golang
const trackableFalse = false
```

Reply:
Done